packages/react-components/react-tabs/src/stories/TabListWithOverflow.stories.tsx

```diff
@@ -269,10 +269,12 @@ export const WithOverflow = () => {
 WithOverflow.parameters = {
   docs: {
     description: {
-      story:
-        'A tab list can support overflow by using Overflow and OverflowItem.' +
-        'NOTE: when adding custom buttons to a tablist -- e.g. the overflow button in this example --' +
-        '`role="tab"` must be manually added for screen reader accessibility.',
+      story: [
+        'A tab list can support overflow by using Overflow and OverflowItem.',
+        '',
+        '**Note**: when adding custom buttons to a tablist, e.g. the overflow button in this example,' +
```

Comment:
_I would even do not notice that there is a note_

### Before

![image](https://user-images.githubusercontent.com/14183168/170065133-7868c342-9f23-47bb-8e2a-f16d63241bd4.png)


### After

![image](https://user-images.githubusercontent.com/14183168/170065024-f7ced2eb-614c-4e6f-bb0e-8ebbde0c3bdc.png)